compiler/env/OMRCompilerEnv.cpp

```diff
@@ -52,6 +52,8 @@ OMR::CompilerEnv::initialize()
 
    self()->initializeTargetEnvironment();
 
+   self()->initializeRelocatableTargetEnvironment();
```

Comment:
This should be called only if we can do AOT compilations.

Reply:
The AOT compilation check is currently done in the TR::Compilation constructor (in OpenJ9, as OMR doesn't support AOT). In the TR::Compilation constructor, if we know it's a AOT compilation then we switch to use relocatableTarget otherwise we use target.
It's initialized here but it may or may not be used depending on AOT compilations.

Reply:
`OMR::CompilerEnv` is a singleton, so it's sort of the master copy of all the potential target environments we may have in the future. The compilation object picks one (ie caches a reference to one) depending on what kind of compilation has been issued.